api/src/main/java/io/kafbat/ui/model/rbac/permission/TopicAction.java

```diff
@@ -13,6 +13,8 @@ public enum TopicAction implements PermissibleAction {
   MESSAGES_READ(VIEW),
   MESSAGES_PRODUCE(VIEW),
   MESSAGES_DELETE(VIEW, EDIT),
+  ANALYSIS_VIEW(VIEW),
```

Comment:
This one is unused atm, we'd need to add this to both backend and frontend

Reply:
Couldn't find where to add it in the frontend.

Reply:
Statistics tab should be disabled if there are no corresponding permissions

Reply:
@Haarolean PR seems ready, can you have a look?